api/docgen/docgen.go

```diff
@@ -414,6 +414,42 @@ func init() {
 		VerifiedAllocationKey: nil,
 		Notify:                nil,
 	})
+
+	addExample(&types.ActorEventBlock{
+		Codec: 0x51,
+		Value: []byte("data"),
+	})
+
+	addExample(&types.ActorEventFilter{
+		Addresses: []address.Address{addr},
+		Fields: map[string][]types.ActorEventBlock{
+			"abc": {
+				{
+					Codec: 0x51,
+					Value: []byte("ddata"),
+				},
+			},
```

Comment:
I'm not convinced this is a great API; It feels really clunky to have to specify the codec here, which is encouraged by the clunky nature of the events themselves of course (!).

We do have the option of leaving the Codec off and just keeping it `map[string][][]byte` and matching on that, the bytes are going to remain the same, identical to the `Value` they provide here. It will lack the precision of saying "this value, which is _this codec_", but that's not too much different from just "this value".

It's not like we're doing anything fancy with the `Codec` here of course—e.g. decoding their `Value` and re-encoding it to the codec specified by the event, the `Value` is the same.

Reply:
@rvagg 

Yeah this is true. I think just a nice way for clients to ensure they're using the right codec as we use different ones for smart contract vs built-in Actor events so they're not left wondering why their filter isn't matching anything. I think best to keep it for now to make it explicit. We can remove this later once our experimental API gets some feedback.